libedit/readline.c

```diff
@@ -48,8 +48,11 @@ __RCSID("$NetBSD: readline.c,v 1.138 2016/09/01 13:23:44 mbalmer Exp $");
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
```

Comment:
The __unix__ macro is also defined on linux, so I would suggest changing the MacOS detection to
#if defined (defined(__APPLE__) && defined(__MACH__))

Currently it breaks the linux build.